### DISCUSSION TITLE
Add on does not show in Add on menu after installing it

### DISCUSSION THREAD

**Discussion by maxbobse** [ScanMountGoat/ldr_tools_blender#28] [Q&A] [answered]
I have installed the add-on and can see the message

```
Modules Installed () from '/Users/max/Downloads/ldr_tools_blender-main.zip' into '/Users/max/Library/Application Support/Blender/4.1/scripts/addons'
```

but when I check the add on menu afterwards I cannot find it anywhere. Scrolled through the list manually and also searched for ldr but nothing ... 

Blender is 
Version 4.1.0 (4.1.0 2024-03-26)

on macOS 14.3.1

Also restarting Blender did not make a difference, still not showing up.

**Comment by ScanMountGoat** [comment, marked as answer]
It looks like you may have downloaded the source code instead of the release. You'll want to download the latest version from releases for either intel or apple silicon depending on your device.
https://github.com/ScanMountGoat/ldr_tools_blender?tab=readme-ov-file#getting-started

**Reply by maxbobse** [reply]
Yeah, sorry .... just noticed it myself - sorry 🤦‍♂️
